crates/red_knot_workspace/tests/check.rs

```diff
@@ -65,6 +65,17 @@ impl<'db> PullTypesVisitor<'db> {
             model: SemanticModel::new(db, file),
         }
     }
+
+    fn visit_assign_target(&mut self, target: &Expr) {
+        match target {
+            Expr::List(ast::ExprList { elts, .. }) | Expr::Tuple(ast::ExprTuple { elts, .. }) => {
+                for element in elts {
+                    self.visit_assign_target(element);
+                }
+            }
+            _ => self.visit_expr(target),
+        }
+    }
```

Comment:
The reason this is required is because in a code like `(a, b) = (1, 2)`, the LHS expression is not available.

Reply:
Since the RHS is a standalone expression whose type should be a cached Salsa query, I think we _could_ query the type of that expression one more time and set a type on the entire LHS expression. But I'm not sure there's any value to doing that; I'm happy with this change to our coverage goal.

Reply:
Yeah, I agree. As you've mentioned, I'm also unsure of what the benefit could be of getting the type of the LHS expression which is either a tuple or list.